app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreenState.kt

```diff
@@ -63,40 +62,39 @@ fun rememberConversationScreenState(
 }
 
 // todo: pass directly the strings, to avoid passing the context
-@OptIn(ExperimentalMaterialApi::class)
 class ConversationScreenState(
     val context: Context,
     val clipboardManager: ClipboardManager,
     val snackBarHostState: SnackbarHostState,
-    val modalBottomSheetState: ModalBottomSheetState,
+    val modalBottomSheetState: WireModalSheetState,
     val coroutineScope: CoroutineScope
 ) {
 
     var bottomSheetMenuType: BottomSheetMenuType by mutableStateOf(BottomSheetMenuType.None)
 
     fun showEditContextMenu(message: UIMessage.Regular) {
         bottomSheetMenuType = BottomSheetMenuType.Edit(message)
-        coroutineScope.launch { modalBottomSheetState.animateTo(ModalBottomSheetValue.Expanded) }
+        coroutineScope.launch { modalBottomSheetState.show() }
     }
 
     fun hideContextMenu(onComplete: () -> Unit = {}) {
         coroutineScope.launch {
-            modalBottomSheetState.animateTo(ModalBottomSheetValue.Hidden)
+            modalBottomSheetState.hide()
             onComplete()
         }
     }
 
     fun copyMessage(text: String) {
         clipboardManager.setText(AnnotatedString(text))
         coroutineScope.launch {
-            modalBottomSheetState.animateTo(ModalBottomSheetValue.Hidden)
+            modalBottomSheetState.hide()
```

Comment:
You are removing animation here as well

Reply:
`animateTo` is `internal` in material3 but we have `hide`, `show`, `expand` and `partialExpand` functions, all of which execute `animateTo` under the hood 😃